contents/docs/llm-analytics/privacy-mode.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Privacy mode
+---
+
+To avoid storing potentially sensitive prompt and completion data, you can enable privacy mode. This excludes the `$ai_input` and `$ai_output_choices` properties from being captured.
```

Comment:
We should include what's captured with/without this mode.

We should also reassure that things like their API tokens are not proxied to reach PostHog servers